app/components/works/embargo_component.rb

```diff
@@ -53,5 +53,17 @@ def embargo_month
     def embargo_day
       embargo_date&.day
     end
+
+    def collection_access_restriction
+      return unless reform.collection_id
```

Comment:
I'm not sure I understand this check. Every item should have a collection ID

Reply:
I believe when I first ran the specs there are instances where it is nil that threw errors. I will verify this morning.